tradebot/data/performance.py

```
@@ -0,0 +1,314 @@
+        Returns:
+            Benchmark results with timing statistics
+        """
+        try:
+            self.logger.info(
+                f"Benchmarking '{operation_name}' over {iterations} iterations"
+            )
+
+            times = []
+            for i in range(iterations):
+                start_time = time.time()
+                result = operation()
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)